entrypoint.sh

```diff
@@ -5,6 +5,9 @@ if test -n "$S3_BUCKET" ; then
 var s3_bucketName = '${S3_BUCKET}';
```

Comment:
The path `/usr/share/nginx/html/catalog_diff/` does not exist. The correct location for the file is: `/usr/share/nginx/html/s3_credentials.js`

Reply:
i updated the location of the file. thx for the hint.